src/Tests/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -2324,70 +2324,20 @@ protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.
 #pragma warning restore 1591
 ");
 
-            var updatedText = new TestAdditionalText("Pages/Index.razor", SourceText.From(@"<h1>Hello world 1</h1>", Encoding.UTF8));
-            driver = driver.ReplaceAdditionalText(additionalTexts.First(f => f.Path == updatedText.Path), updatedText);
-
             // now run the generator with suppression
             var suppressedOptions = optionsProvider.Clone();
             suppressedOptions.TestGlobalOptions["build_property.SuppressRazorSourceGenerator"] = "true";
             driver = driver.WithUpdatedAnalyzerConfigOptions(suppressedOptions);
 
-            // results should be the same (even though we changed text)
-            result = RunGenerator(compilation!, ref driver)
-                    .VerifyOutputsMatch(result);
+            // results should be empty
+            var emptyResult = RunGenerator(compilation!, ref driver)
+                    .VerifyPageOutput();
 
             // now unsuppress and re-run
             driver = driver.WithUpdatedAnalyzerConfigOptions(optionsProvider);
 
             result = RunGenerator(compilation!, ref driver)
-                .VerifyPageOutput(
-@"#pragma checksum ""Pages/Index.razor"" ""{ff1816ec-aa5e-4d10-87f7-6f4963833460}"" ""9845f29d52d0de05d908b1afbab2e2325932b678""
-// <auto-generated/>
-#pragma warning disable 1591
-namespace MyApp.Pages
-{
-    #line hidden
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Threading.Tasks;
-    using Microsoft.AspNetCore.Components;
-    public partial class Index : global::Microsoft.AspNetCore.Components.ComponentBase
-    {
-        #pragma warning disable 1998
-        protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
-        {
-            __builder.AddMarkupContent(0, ""<h1>Hello world 1</h1>"");
-        }
-        #pragma warning restore 1998
-    }
-}
-#pragma warning restore 1591
-",
-@"#pragma checksum ""Pages/Counter.razor"" ""{ff1816ec-aa5e-4d10-87f7-6f4963833460}"" ""0de17e526cd536d59072aa0e924e99111b16b97a""
-// <auto-generated/>
-#pragma warning disable 1591
-namespace MyApp.Pages
-{
-    #line hidden
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Threading.Tasks;
-    using Microsoft.AspNetCore.Components;
-    public partial class Counter : global::Microsoft.AspNetCore.Components.ComponentBase
-    {
-        #pragma warning disable 1998
-        protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
-        {
-            __builder.AddMarkupContent(0, ""<h1>Counter</h1>"");
-        }
-        #pragma warning restore 1998
-    }
-}
-#pragma warning restore 1591
-"
-);
+                .VerifyOutputsMatch(result);
```

Comment:
nit: I find it a tiny bit confusing to read and assign 'result' here, particularly since it doesn't look like 'result' is read again after this.